modules/workbench/modules/reporting/assets/schemas/user.json

```diff
@@ -103,13 +103,31 @@
   {
     "name": "ras_login_gov_bypass_time",
     "type": "TIMESTAMP",
-    "description": "Time an administrator bypassed the RAS Login.gov requirement for this user.",
+    "description": "[DELETED] Was: Time an administrator bypassed the RAS Login.gov requirement for this user.",
     "mode": "NULLABLE"
   },
   {
     "name": "ras_login_gov_completion_time",
     "type": "TIMESTAMP",
-    "description": "Time user completed the RAS Login.gov account link.",
+    "description": "[DELETED] Was: Time user completed the RAS Login.gov account link.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "identity_bypass_time",
+    "type": "TIMESTAMP",
+    "description": "Time an administrator bypassed the identity requirement for this user.",
```

Comment:
These look reasonable but I think we can improve the wording a bit... "when" instead of "time" maybe?

Reply:
I think your suggestion sounds reasonable, but all the other module bypass and completion fields use "time". Is it ok to update them all to "when"?